bakery/checkers/operations.go

```diff
@@ -14,6 +14,8 @@ type opKey struct{}
 // given operations. An allow caveat will succeed only if one of the allowed
 // operations is in ops; a deny caveat will succeed only if none of the denied
 // operations are in ops.
+//
+// This uses the same context key as ContextWithOpAuthorizer.
```

Comment:
HAven't I complained about this commend once?

Reply:
Yeah, it snuck back in.